source/api_documentation/network_integration/advertiser_campaigns/_get_advertiser_campaign_unarchive.rst

```diff
@@ -0,0 +1,15 @@
+.. container:: endpoint-long-description
```

Comment:
I would like to remove the GET action for archive from the API endpoint.

On Tue, Mar 8, 2016 at 4:41 AM, Zsolt Mathe <notifications@github.com>
wrote:

> In
> source/api_documentation/network_integration/advertiser_campaigns/_get_advertiser_campaign_unarchive.rst
> <https://github.com/Invoca/developer-docs/pull/21#discussion_r55353596>:
>
> > @@ -0,0 +1,15 @@
> > +.. container:: endpoint-long-description
>
> Yes, I agree with you. I just replicated the examples above with go_live,
> where there's also a GET and a POST example. I think we should remove the
> GET example for the archive action. Would be ideal to remove the GET for
> go_live as well, but that's not possible without a version bump.
> @mikeweaver <https://github.com/mikeweaver>, @spencerpbartz
> <https://github.com/spencerpbartz> Do we also want to update the API
> endpoint to not allow GET request? This would be safe for the archive
> action since that wasn't exposed in the API doc till now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Invoca/developer-docs/pull/21/files#r55353596>.
>



-- 
<http://www.invoca.com/>

*Mike Weaver*

Director of Software Engineering

m:| (805) 448-8900 <8054488900>

p: | (805) 880-7711 <8058807711>

skype: | mike.r.weaver


[image: Invoca Named to Inc. 500 List of Fastest-Growing Private Companies]
<http://www.invoca.com/about/press/invoca-named-inc-500-list-fastest-growing-private-companies/>
Invoca
Named to Inc. 500 List of Fastest-Growing Private Companies
<http://www.invoca.com/about/press/invoca-named-inc-500-list-fastest-growing-private-companies/>